bigbluebutton-html5/imports/ui/components/video-provider/component.jsx

```diff
@@ -262,11 +266,11 @@ class VideoProvider extends Component {
         break;
 
       case 'iceCandidate':
-        this.handleIceCandidate(parsedMessage);
+        //this.handleIceCandidate(parsedMessage);
```

Comment:
It seems you've forgotten to unblock the ICE handling from SFU.

Reply:
Sorry, my bad. Fixing now.

Reply:
Fixed.